licence-checker.hjson

```diff
@@ -10,6 +10,7 @@
     Licensed under the Apache License, Version 2.0, see LICENSE for details.
     SPDX-License-Identifier: Apache-2.0
     ''',
+  match_regex: "false",
```

Comment:
Can we just use `match_regex: false` here? (Then it will parse as a bool, rather than as a string that happens to look like a bool). Sanity check:
```
>>> import hjson
>>> hjson.loads('{ foo: true, bar: "true" }')
OrderedDict([('foo', True), ('bar', 'true')])
```
Let's go with the `foo` option!

Reply:
I was sticking to your hjson reference manual which asks for all values to be in quotes (https://github.com/lowRISC/opentitan/blob/master/doc/rm/hjson_usage_style.md#simple-key-value-entries). Happy to change that.

Reply:
Oh, wow. I didn't realise that we mandated that. How silly. Ok, well I guess we should be consistent. Eugh.